packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -38,7 +38,7 @@ describe("publish", () => {
 
   describe("environments", () => {
     it("should use legacy environments by default", async () => {
-      writeWranglerToml();
+      writeWranglerToml({ env: { "some-env": {} } });
```

Comment:
The addition of this env config is to avoid getting the additional warning message about there being no matching environment to that passed into the `wrangler publish` command.